library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -36,6 +38,14 @@ sealed class Conversation {
             }
         }
 
+    val messageExpiration: MessageExpiration
```

Comment:
suggestion: we used message disappearingSettings everywhere, I would suggest let's have the same naming here too. wdyt?